src/sidebar/components/FilterSelect.tsx

```diff
@@ -47,9 +47,11 @@ function FilterSelect({
       label={menuLabel}
       title={title}
       contentClass={classnames(
-        // Don't let filter list get too terribly tall. On shorter screens,
-        // restrict to 70vh; set a static max-height for taller screens.
-        'max-h-[70vh] tall:max-h-[504px] overflow-y-auto'
+        // Don't let filter list get too terribly tall. Where space allows,
+        // use `504px` (this odd number is to maximize the likelihood that the
+        // overflow cutoff is in the middle of an option, making it more obvious
+        // that the list is scrollable), but don't exceed 60vh.
```

Comment:
Per your footnote, using a drop-shadow here would be nicer than relying on the item size, as that may be brittle in the face of design changes or larger font sizes.

Reply:
I agree. The `60vh` max-height guardrail is also coupled to the location of this component within the UI, which is problematic.